testutil/integration/model_test.go

```diff
@@ -18,73 +28,127 @@ import (
 var _ = Describe("Model Registry", func() {
 	const timeout = time.Second * 5
 	const interval = time.Second * 1
+	type Format string
+
+	const (
+		FormatSavedModel  Format = "SavedModel"
+		FormatONNX        Format = "ONNX"
+		FormatH5          Format = "H5"
+		FormatPMML        Format = "PMML"
+		FormatCaffeModel  Format = "CaffeModel"
+		FormatNetDef      Format = "NetDef"
+		FormatMXNETParams Format = "MXNETParams"
+		FormatTorchScript Format = "TorchScript"
+		FormatGraphDef    Format = "GraphDef"
+		FormatTensorRT    Format = "TensorRT"
+		FormatSKLearn     Format = "SKLearn"
+		FormatXGBoost     Format = "XGBoost"
+		FormatMLflow      Format = "MLflow"
+		FormatMLlib       Format = "MLlib"
+	)
+
+	var ModelExtract = []struct {
```

Comment:
```suggestion
	var modelExtract = []struct {
```
Lowercase  is needed when no external package reference

Reply:
OK